javascript/packages/utils/src/tableCli.ts

```diff
@@ -96,12 +120,26 @@ export class CreateLogTable {
               input[index] = inp;
             }
           });
-        this.table!.push(input);
+        if (logType === "text") {
+          if (input[0] === "\x1B[36mCommand\x1B[0m") {
+            input[1] = input[1].replace(/\n/g, " ");
+          }
+          // if input has a JSON - that means a merged cell
+          if (input[0]?.content) {
+            input[0] = input[0]?.content;
+          }
+          console.log(input.join(" : "));
+        } else if (logType === "silent") {
+          return;
+        } else if (logType === "table") {
+          this.table!.push(input);
+        }
       });
   };
 
   print = () => {
-    if (!silent) console.log(this.table!.toString());
+    if (logType === "silent" || logType === "text") return;
```

Comment:
```suggestion
```
I think we don't need this, since we are `only` printing if the `logType` is `table`.